src/client/package.json

```diff
@@ -22,7 +22,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "npm run dev",
-    "lint": "eslint src",
+    "dev:local": "npm run dev -- --endpoint local",
```

Comment:
This is to specify the server backend you're building against, ie. local or cloud